_posts/2014-03-04-companies.html

```diff
@@ -114,12 +139,12 @@ <h3 id="associated-resources">Associated Resources</h3>
     <tr>
       <td class="nowrap"><code class="redtext">array</code></td>
       <td><code class="bluetext">contacts()</code></td>
-      <td>First Ten Contacts associated with this company</td>
+      <td>Contacts associated with this company</td>
     </tr>
     <tr>
       <td><code class="redtext">array</code></td>
       <td class="nowrap"><code class="bluetext">pending_deals()</code></td>
-      <td class="nowrap">Most recently updated pending deals associated with this company (limited to 10).</td>
+      <td class="nowrap">Most recently updated pending deals associated with this company.</td>
```

Comment:
@elanovasconcelos Think this is actually deals with the earliest expected close date no? Don't think it's to do with `modified_at` date. 

cc/ @lorobode Can you check this??